static/js/reboot.js

```diff
@@ -0,0 +1,20 @@
+function reboot() {
+  var xmlHttp = new XMLHttpRequest();
+  xmlHttp.open('POST', '/api/reboot', true);
+  xmlHttp.setRequestHeader("Authorization", "Basic "+btoa("admin:feathers"))
+  xmlHttp.onload = async function() {
+    if (xmlHttp.status == 200) {
+      alert("device will reboot shortly. Please go back to salt to look for the device again")
```

Comment:
I don't think we should be mentioning salt here. Our users won't know what Salt is.